content/help/system-requirements.adoc

```diff
@@ -153,6 +153,9 @@ the following window managers:
 * i3 (Arch Linux)
 * Unity (Ubuntu prior to 18.04)
 
+In case you use MATE (not officially supported) and find rendering problems and keys that do not
```

Comment:
We don't call out problems this specifically on the system requirements page. We provide the link above that already redirects to the known issues page.

Reply:
But for example, you do that for AMD above: ** AMD/ATI: should work any modern Radeon GPU, but check for some issues here:
   link:/help/known-system-related-issues/#_bad_graphics_cards[Bad Graphics Cards]

Reply:
I have condensed it now.

Reply:
That text didn't call out specific issues though. The problem with saying specific issues here is that this text could get out of sync with the Known issues page very easily. Keeping it generic here and referring to the other page is better in the long run. The updated text looks fine to me now.